cps/CP1/CP1.md

```diff
@@ -22,9 +22,54 @@ The next step is the funding process, which will be an on-chain Treasury Proposa
 | --- | --- | :---: |
 |5|Create an on-chain Treasury proposal|-|
 
+### TEMPLATE
+```
```

Comment:
I wanted to frame the template in a way but couldn't figure out how to do it so I used ``` ... ```